example/android/app/src/main/java/com/klaviyoreactnativesdkexample/MainApplication.kt

```diff
@@ -10,19 +10,21 @@ import com.facebook.react.ReactPackage
 import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint.load
 import com.facebook.react.defaults.DefaultReactHost.getDefaultReactHost
 import com.facebook.react.defaults.DefaultReactNativeHost
-import com.facebook.react.flipper.ReactNativeFlipper
+import com.facebook.react.soloader.OpenSourceMergedSoMapping
```

Comment:
I know they removed flipper support but what is soloader?

Reply:
https://github.com/facebook/SoLoader
Some kind of recursive library loader. Don't think it is related to flipper, just happens to be close in the alphabet for import order.

Reply:
which is funny because that explicitly says its needed for SDKs 23 and below, so IDK why they still have it in the template project if they bumped support to 24+.